app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -150,6 +150,8 @@ def textual_allvms_size
   end
 
   def textual_total_miq_templates
+    return nil if @record.kind_of?(EmsClusterOpenstackInfra)
```

Comment:
I would much prefer to ask the record if it supports some feature/operation and update UI based on that, instead of adding that knowledge to the helper. This would make the code more resilient to future changes.